openslides/users/migrations/0002_auto_20150630_0143.py

```diff
@@ -0,0 +1,28 @@
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
This is dangerous. If a user migrates database but does not upgrade to Django 1.8 he might run into trouble. I this case it looks ok but we should think about dropping old Django support in future cases.